### ISSUE TITLE
model: pytorch: base: Update calls to numpy array creation

### ISSUE THREAD

**Issue by pdxjohnny** [intel/dffml#994]
```
/home/runner/work/dffml/dffml/model/pytorch/dffml_model_pytorch/pytorch_base.py:181: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray
  x_cols[feature] = np.array(x_cols[feature])
```

**Comment by iamapickle** [comment]
Do you want the warning to be removed simply, since result will still remain the same? or to change the dtype to object?
and can i do this?

**Comment by pdxjohnny** [comment]
@iamapickle yes the issue is up for grabs, see https://intel.github.io/dffml/master/contributing/dev_community.html#who-s-working-on-what

I'm not sure. I haven't looked into it. The numpy docs or source relevant to this deprecation warning should be read to understand what our code should be changed to.

**Comment by Pratikrocks** [comment]
Hi @iamandeepsandhu are you taking up this issue, if not please inform me I will be taking it :)

**Comment by iamapickle** [comment]
#1000

**Comment by praneethratna** [comment]
@pdxjohnny Hi , I'm new to open source contribution and if this is a good first issue could this be assigned to me as i would like to work on it?

**Comment by snnipetr** [comment]
@pdxjohnny  I think this issue still persists. Can I take up this one ?

**Comment by programmer290399** [comment]
Hey @snnipetr !!

Yess!! If you can reproduce the issue then please go ahead...
So as I can see there has been some work around this in #1000, so please take a look at that.
 
Also, as a rule, we don't ask if we can do something, we state our intention to try to do it, then follow [these](https://intel.github.io/dffml/master/contributing/dev_community.html#who-s-working-on-what) guidelines.

Before making a PR please also take a look at the relevant documentation up [here](https://intel.github.io/dffml/master/contributing/git.html#issue-and-pull-request-title-formatting)

I hope this helps, please feel free to ask any questions on gitter :)
